src/exchangeRateRouter.ts

```diff
@@ -301,6 +310,108 @@ const sendExchangeRates: express.RequestHandler = (req, res, next): void => {
   res.json({ data: exReq.requestedRatesResult.data })
 }
 
+const sendCoinranks: express.RequestHandler = async (
+  req,
+  res,
+  next
+): Promise<void> => {
+  const { ratesServerAddress } = config
+  const now = new Date()
+  const nowTimestamp = now.getTime()
+
+  const exReq = req as ExpressRequest
+  if (exReq == null) return next(500)
+
+  let query: CoinrankReq
+  try {
+    query = asCoinrankReq(req.query)
+  } catch (e) {
+    res
+      .status(400)
+      .send(`Error: ${e instanceof Error ? e.message : 'Unknown error'}`)
+    return
+  }
+  const { fiatCode, start, length } = query
+
+  try {
+    if (start < 1 || start > 2000) {
+      res
+        .status(400)
+        .send(`Invalid start param: ${start}. Must be between 1-2000`)
+      return
+    }
+    if (length < 1 || length > 100) {
+      res
+        .status(400)
+        .send(`Invalid length param: ${length}. Must be between 1-100`)
+      return
+    }
+    const fiatSuffix = fiatCode.split(':')[1]
```

Comment:
Nitpick, just use the fiatCode as is for the key. And replace any hardcoded USD/iso:USD value with the default fiat from the config

Reply:
ok. note there are other hardcoded `iso:USD` occurences in the file